src/main/java/com/synopsys/integration/jenkins/detect/service/strategy/DetectScriptStrategy.java

```diff
@@ -35,10 +35,10 @@
 
 public class DetectScriptStrategy extends DetectExecutionStrategy {
     public static final String DETECT_INSTALL_DIRECTORY = "Detect_Installation";
-    public static final String SUPPORTED_SHELL_SCRIPT_URL = "https://detect.synopsys.com/detect7.sh";
-    public static final String SHELL_SCRIPT_FILENAME = "detect7.sh";
-    public static final String SUPPORTED_POWERSHELL_SCRIPT_URL = "https://detect.synopsys.com/detect7.ps1";
-    public static final String POWERSHELL_SCRIPT_FILENAME = "detect7.ps1";
+    public static final String SUPPORTED_SHELL_SCRIPT_URL = "https://detect.synopsys.com/detect8.sh";
+    public static final String SHELL_SCRIPT_FILENAME = "detect8.sh";
+    public static final String SUPPORTED_POWERSHELL_SCRIPT_URL = "https://detect.synopsys.com/detect8.ps1";
+    public static final String POWERSHELL_SCRIPT_FILENAME = "detect8.ps1";
 
```

Comment:
Certainly not a big deal, but maybe next time someone works on this we could move to a single int representing the Detect major version that gets used in 3 places (feels like someone could overlook one of the three places if not careful).

Reply:
(Sorry, just now realized that this PR has been waiting for my review for 6 weeks... I'm a bit late with this feedback!)